Assets/Scripts/P2PNetcodeSample/Networking/EOSTransport.cs

```diff
@@ -103,19 +103,19 @@ public class EOSTransport : NetworkTransport
         private Queue<Tuple<ProductUserId, ulong, bool>> ConnectedDisconnectedUserEvents = null;
 
         [System.Diagnostics.Conditional("EOS_TRANSPORT_DEBUG")]
-        private void print(string msg)
+        private void log(string msg)
         {
             Debug.Log(msg);
         }
 
         [System.Diagnostics.Conditional("EOS_TRANSPORT_DEBUG")]
-        private void printWarning(string msg)
+        private void logWarning(string msg)
```

Comment:
Changing this to "Title Case" would probably be more consistent with our naming conventions (so it would be `LogWarning`).